src/furo/assets/styles/_scaffold.sass

```diff
@@ -50,8 +50,8 @@ article
 
 .page
   display: flex
-  // fill the viewport for pages with little content.
-  min-height: 100%
+  height: 100vh
+  overflow-y: hidden
```

Comment:
Hmm... I'm hesitant to ever use overflow `hidden` since it inherently means clipping content if something about the styles/content is wrong and I'd much rather enforce correctness via other means here.